src/rawchannel.cc

```diff
@@ -122,9 +126,18 @@ class RawChannel : public Nan::ObjectWrap
       if (ioctl(m_SocketFd, SIOCGIFINDEX, &ifr) != 0)
         goto on_error;
 
+      // Configuration updated to use the CAN_FD 
       err_mask = CAN_ERR_MASK;
-      if (setsockopt(m_SocketFd, SOL_CAN_RAW, CAN_RAW_ERR_FILTER, &err_mask, sizeof(err_mask)) != 0)
-        goto on_error;
+      Flag_CANFD_Used = 1 ;                                                                               // try to use CAN_FD first
+      if (setsockopt(m_SocketFd, SOL_CAN_RAW, CAN_RAW_FD_FRAMES,&canfd_on, sizeof(canfd_on)) != 0)        // configuration for CAN_FD
```

Comment:
missing whitespace between _RAMES,&canfd

and the comment should be: switch socket to CANFD mode

Reply:
done